Competition2018/src/main/java/competition/Robot.java

```diff
@@ -17,6 +18,9 @@
 public class Robot extends BaseRobot {
 
     static Logger log = Logger.getLogger(Robot.class);
+    
+    AutonomousCommandSelector autonomousCommandSelector;
```

Comment:
this needs to be initialized... at the moment it will always be `null`. Same with `drive`.

Reply:
@johnli01 See this line in the 2017 code:
https://github.com/Team488/TeamXbot2017/blob/42ae671588f1e2083a7c3e96e5f44d092c775966/Competition2017/src/competition/Robot.java#L30